app/views/releases/index.html.erb

```diff
@@ -31,11 +27,19 @@
       </table>
 
       <%= paginate @pagy %>
+
+      <p class="pull-right">
+        <%= link_to "Create release manually", new_project_release_path(@project), class: "btn btn-default" %>
+      </p>
     <% elsif @project.release_branch.present? %>
       <div class="alert alert-warning">
         <p>No release created from release branch <%= @project.release_branch %>.</p>
         <p>Add a webhook to samson from your CI to trigger releases on green builds or from your Code host to get releases on merge.</p>
         <p>Samsons Github user needs write permisson to the repo to create tags.</p>
+
+        <p>
+          <%= link_to "Create release manually", new_project_release_path(@project), class: "btn btn-default" %>
+        </p>
       </div>
```

Comment:
how about something like `<% create_button = capture do %>` and then reuse that instead of copying the button ?

Reply:
I honestly think that would be a lot more confusing than just having two copies. This isn't business logic, it's declarative UI code...